src/Library/demos/Datepicker/index.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env gjs
+
+const {Gtk} = imports.gi;
+const DatePickerDemo = require('./main.js').DatePickerDemo;
```

Comment:
GJS is not Node.js, there is no such thing as `require`.

This cannot work. Did you test your code at all?